build.sbt

```diff
@@ -75,3 +70,15 @@ lazy val checkerPlugin = (projectMatrix in file("checker-plugin"))
   .settings(libraryDependencies ++= Seq(akkaTyped % Test, akkaPersistence % Test, akkaProjections % Test))
   .dependsOn(checkerLibrary)
   .jvmPlatform(scalaVersions = supportedScalaVersions)
+
+lazy val schemaDumpPlugin = (projectMatrix in file("sbt-dumpschema"))
+  .enablePlugins(SbtPlugin)
+  .settings(name := "sbt-dumpschema")
+  .settings(commonSettings)
+  .settings(pluginCrossBuild / sbtVersion :=
+    virtualAxes.value
+      .collectFirst { case x: ScalaVersionAxis => x.value }
+      .map { case "2.12" => "1.2.8" }
+      .getOrElse("1.5.2"))
+  .dependsOn(checkerLibrary)
+  .jvmPlatform(scalaVersions = Seq(scalaVersion212))
```

Comment:
Hmmm does `getOrElse` from line 82 even matter, given that there isn't really a cross-version build support, but instead just Scala 2.12 is supported?